frontend/src/hooks/useMessageValidation.ts

```diff
@@ -0,0 +1,38 @@
+import { useFormik } from "formik";
+import * as Yup from "yup";
+
+const regExp =
+    /^(https?):\/\/(?:www\.)?[a-zA-Z]{2,}\.[a-z]{2,5}(?!.*(\/{2}|https?|:|\.| )).*$/;
+export const maxCount = 500;
+
+interface IMessageVal {
+    handleSubmitForm: (content: string) => void;
+    defaultContent?: string;
+}
+
+const useMessageValidation = ({
+    handleSubmitForm,
+    defaultContent
+}: IMessageVal) => {
+    const isImage = false;
+    const formik = useFormik({
+        initialValues: {
+            content: defaultContent ? defaultContent : ""
```

Comment:
This if is unnecessary, cause if "defaultContent" is empty, this empty value will be placed here.

Reply:
No, It return undefined

Reply:
ok, in that case